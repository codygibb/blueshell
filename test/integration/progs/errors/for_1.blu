func f() {
	return func() {
		return (5, "not-bool")
	}
}

for x in f() {
	print x
}
