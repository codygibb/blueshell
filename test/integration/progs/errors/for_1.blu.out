error: Incorrect_type (iter, (int, str), (prim, bool))
